src/qibo/models/parameter.py

```diff
@@ -0,0 +1,42 @@
+"""Model for combining trainable parameters and possible features into circuit parameters."""
+
+
+class Parameter:
+    def __init__(self, func, variational_parameters, features=None):
+        self._variational_parameters = variational_parameters
+        self._featurep = features
+        self.nparams = len(variational_parameters)
+        self.lambdaf = func
+
+    def _apply_func(self, fixed_params=None):
+        params = []
+        if self._featurep is not None:
+            params.append(self._featurep)
+        if fixed_params:
+            params.extend(fixed_params)
```

Comment:
Can you please explain what you mean with `fixed_params`?

Reply:
If you have a gate parameter $\theta = 10\theta_i + 20$, the fixed_params returns 20. It is basically the invariant part of the lambda function, with regards to the parameter we are considering (here $\theta_i$)